pwndbg/heap/ptmalloc.py

```diff
@@ -49,8 +67,8 @@ def __init__(self, addr):
         self._bk = None
         self._fd_nextsize = None
         self._bk_nextsize = None
-
-        # TODO key, REVEAL_PTR
+        self._arena = arena
+        self._is_top_chunk = None
```

Comment:
Minor point, we should probably default boolean fields to `True` or `False`, and in this case `False`. It will help with type checking later.

Reply:
I'm not sure about that, with the current approach we will initialize a given field on its first use